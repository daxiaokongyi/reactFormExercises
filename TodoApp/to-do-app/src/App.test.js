import { render } from '@testing-library/react';
import App from './App';

// smoke test
test('renders without crashing', () => {
  render(<App />);
});

// snapshot test
test('matches snapshots', () => {
  const {asFragment} = render(<App/>);
  expect(asFragment()).toMatchSnapshot();
})
