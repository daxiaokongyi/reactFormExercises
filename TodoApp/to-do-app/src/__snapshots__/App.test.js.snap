// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshots 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <div>
        <form>
          <label
            for="todo"
          >
            New Todo: 
          </label>
          <input
            id="todo"
            name="todo"
            type="text"
            value=""
          />
          <button>
            Add a Todo
          </button>
        </form>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
